doc/sphinx/README.rst

```diff
@@ -377,6 +401,56 @@ DON'T
 
      .. tacv:: assert form as intro_pattern
 
+Using the ``.. coqtop::`` directive for syntax highlighting
+-----------------------------------------------------------
+
+DO
+  .. code::
+
+     A tactic of the form:
+
+     .. coqdoc::
+
+        do [ t1 | … | tn ].
```

Comment:
Note that currently there is a rendering problem (with margins) when using the `.. coqdoc::` directive which encourages people to use `.. coqtop:: in` instead.

Reply:
No sorry, never filled an issue for this. Here is an example though: https://coq.inria.fr/refman/proof-engine/ssreflect-proof-language.html#pattern-assignment (the first block is a coqdoc).

Reply:
OK, I have a patch for this, based on this branch.  I'll post it after we merge this.